drivers/SmartThings/matter-window-covering/profiles/window-covering-tilt-only-battery.yml

```diff
@@ -0,0 +1,21 @@
+name: window-covering-tilt-only-battery
+components:
+- id: main
+  capabilities:
+  - id: windowShade
+    version: 1
+  - id: windowShadePreset
```

Comment:
This capability is currently implemented using `GoToLiftPercentage`, so I think we need to do one of two things:

1)  Only support `windowShadePreset` and the associated `presetPosition` preference if the `windowShadeLevel` capability is supported
2) Allow `windowShadePreset` to be used to set the tilt to a preset value

If we go with Option 2, we'd also have the case where a device supports both `windowShadeLevel` and `windowShadeTiltLevel `. That would get a bit tricky to deal with the setting of the preset position for both level and tilt. We may need a new preference for that case.

Reply:
My feeling is that it might make sense to go with Option 2 and create a new preference for the tilt preset position.

Reply:
I made this change in d5424d31248d3d81549e817624f8905565ec15da, let me know if that looks ok or not. Also, I created it as an embedded preference, is that ok or would we want this to be created as an 'official' preference?